spaceapi_server/Cargo.toml

```diff
@@ -12,6 +12,7 @@ rustc-serialize = "^0.3"
 iron = "^0.2"
 log = "^0.3"
 urlencoded = "^0.2"
+router = "0.0.15"
```

Comment:
Below version 0.1 every version can be breaking, according to semantic versioning. So we pin it to the specific version. Also, semver (the `^`) is default, even if you don't specify it.